tox.ini

```diff
@@ -6,10 +6,11 @@ setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}/multinosetests
 commands =
     make check
-deps =
-    -rrequirements-dev.txt
+deps = -rrequirements.txt
+
 whitelist_externals =
     make
 
 [flake8]
 exclude = tests/test_data/*
+ignore = E731,W503
```

Comment:
This is necessary because latest pep8 version added some warnings, E731 cares about lamda expressions. This need to change existing code.